### PR TITLE
Cryptocurrency: Swap out external endpoint for proxy endpoint.

### PR DESCRIPTION
**Pull request by pjhampton**
## Description of new Instant Answer, or changes

Swaps out the external api url with the internal proxy url
screen shot 2017-06-29 at 10 14 13 am

## Related Issues and Discussions

Fixes: https://github.com/duckduckgo/zeroclickinfo-spice/issues/3349

## People to notify

@moollaza 

---

Instant Answer Page: https://duck.co/ia/view/cryptocurrency

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/spice/cryptocurrency/cryptocurrency.js [hunk 1]
```diff
@@ -101,12 +101,15 @@
                 Converter.fromCurrency = from;
             }
 
-            var endpoint = "https://api.cryptonator.com/api/full/" + from + "-" + to;
-            $.get(endpoint, function(payload) {
-                Converter.rate = parseFloat(payload.ticker.price);
+            var endpoint = "/js/spice/cryptocurrency/cryptonator/" + from + "/" + to;
```
Comment (moollaza) [inline share/spice/cryptocurrency/cryptocurrency.js]: Should be `"/js/spice/cryptonator/"`

Its as if we're creating a separate IA -- its not a subpath

I would name it `cryptocurrency_json`

**Comment by pjhampton** [comment]
Thanks @moollaza 
This should be good to go 👍

**Comment by bsstoner** [comment]
This was released to production 🚀

**Pull request merged**